### ISSUE TITLE
Path in your store, should come from Settings, but not from public part

### ISSUE THREAD

**Issue by mattiLeBlanc** [CollectionFS/Meteor-CollectionFS#815]
Hi,

I was wondering, when I create my store:
```
@XmlFiles = new FS.Collection 'xmlFiles',
  stores: [
    new FS.Store.FileSystem 'xmlFile',
      path: Meteor.settings.public.uploadFolder
      maxTries: 2
    ]
```
in a Meteor project, I want my path to be in the settings.json.
Locally I would use a devSettings.json.
However, I do not want to put the upload path in the public section of the settings, but I have to, otherwise the variable is not available when the Store is initialised in the client.
Then later on, you can see the path in store in the browser.

So in your opinion, doesn't it matter that this path is exposed telling outsiders about your folder structure?

**Comment by nooitaf** [comment]
The filesystem path is server only, so you can just remove it on the client
`````javascript
// server
@XmlFiles = new FS.Collection 'xmlFiles',
  stores: [
    new FS.Store.FileSystem 'xmlFile',
      path: Meteor.settings.public.uploadFolder
      maxTries: 2
    ]

// client
@XmlFiles = new FS.Collection 'xmlFiles',
  stores: [
    new FS.Store.FileSystem 'xmlFile'
    ]
`````

**Comment by mattiLeBlanc** [comment]
Yes, I thought something like this would be possible. I just wasnt sure if you were allowed to define the store twice (but I guess on is server and the other client).
Cheers
